docs/index.rst

```diff
@@ -52,6 +52,7 @@ Learn more about parametrization in the :ref:`Parametrization section <parametri
    optimization.rst
    parametrization.rst
    machinelearning.rst
+   r.rst
```

Comment:
please move it downwards in this list, since it's much more specific than the API reference. I would advise to put it after or before the benchmarking examples

Reply:
Maybe you should not underestimate the number of R users in this world :-) Anyway, I've moved it downwards.